### PR TITLE
adding theme-blue to table

### LINKED ISSUE #405: Table block - add variation - theme-blue
Add a variation to table block for theme-blue
https://www.sunstar.com/sustainability/environment/management/
image

### PR DESCRIPTION
**Pull request by mkbansal1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after), along with a short summary of changes:

## Issue

Fixes #405 

## Changelog:
- Adding theme-blue to table
- Adding highlight-first-column to table

## Test URLs:
- Original: https://www.sunstar.com/sustainability/environment/management/
- Before: https://main--sunstar--hlxsites.hlx.live/sustainability/environment/management
- After: https://405-table-theme-blue--sunstar--hlxsites.hlx.live/sustainability/environment/management

## Library

- [ ] New Blocks introduced in this PR
      If yes, please provide details below
**Block Name** : For e.g. _cards_
- [ ] Documented in sidekick library

- [x] New variations introduced in this PR
**Variation Name** : table (theme-blue), table (highlight-first-column)
- [x] Documented in sidekick library

- [ ] New mixins introduced in this PR
**Mixin Name** :  For e.g. _compact, white_
- [ ] Documented in sidekick library

### CODE REVIEW

**Review by sdmcraft — APPROVED** [review]

File: blocks/table/table.css [hunk 1]
```diff
@@ -183,6 +183,15 @@ main .table.text-table table tr td {
     flex-basis: calc(100% - 7.5em);
 }
 
+/* Blue theme */
+main .table.theme-blue table tbody tr td {
+    color: var(--secondary);
+}
+
+main .table.highlight-first-column table td:nth-child(1) {
```
Comment (shrotia) [inline blocks/table/table.css:191]: this style broke the Table having Merged Cell, let me try adding empty div with (display:none) to handle this.

image

image

Comment (shrotia) [reply blocks/table/table.css:191]: Logged: https://github.com/hlxsites/sunstar/issues/407

**Comment by JiangLong2019** [comment]
Guys, seems we are working on similar issues -_-

https://github.com/hlxsites/sunstar/issues/318

**Comment by shrotia** [comment]
@JiangLong2019  let me mark https://github.com/hlxsites/sunstar/issues/407 as a duplicate of https://github.com/hlxsites/sunstar/issues/318

**Pull request merged**